learn/core_concepts/primary_key.md

```diff
@@ -141,7 +141,7 @@ The code below updates the primary key to `title`:
 
 ### Meilisearch guesses your primary key
 
-Suppose you add documents to an index without previously setting its primary key. In this case, Meilisearch will automatically look for an attribute containing the string `id` in a case-insensitive manner (for example, `uid`, `BookId`, `ID`, `123id123`) in your first document and set it as the index's primary key.
+Suppose you add documents to an index without previously setting its primary key. In this case, Meilisearch will automatically look for an attribute ending with the string `id` in a case-insensitive manner (for example, `uid`, `BookId`, `ID`) in your first document and set it as the index's primary key. If it finds multiple attributes ending with `id`, it will throw an error and ask you to manually set the primary key.
```

Comment:
I wonder if we could describe the error case with multiple attributes maybe in the next paragraph as that would emphasize that the behavior of Meilisearch is similar when there is no candidate and when there are multiple candidates found.

> If Meilisearch cannot find a suitable attribute, or if it finds multiple attributes, ...